### ISSUE TITLE
How to use validation dataset during training?

### ISSUE THREAD

**Issue by Peachypie98** [OpenHelix-Team/VLA-Adapter#41]
Image 1:
<img width="711" height="267" alt="Image" src="https://github.com/user-attachments/assets/c72477bd-4d4b-4a4e-966e-979edea5a33a" />

Image 2 (finetune.py):
<img width="915" height="594" alt="Image" src="https://github.com/user-attachments/assets/1099bd87-a5fd-41b6-bb97-6b6060b4cf5e" />

Hello!

I’d like to ask whether this is the correct way to use a validation set during model training.

Based on the image I attached, to enable validation I only need to include `val.tfrecord` in the dataset configuration and set `cfg.use_val_set = True`? Is that sufficient, or is there an additional step required to properly separate training and validation data?

When I tried this approach, I noticed that `len(dataloader)` and `len(val_dataloader)` return the same value, even though the validation dataset contains more samples than the training dataset. This makes me suspect that the datasets may not actually be split as intended.

Could you clarify whether this setup is correct?

Thank You!
